setup.py

```diff
@@ -1,13 +1,21 @@
-import setuptools
+from setuptools import find_packages, setup
+import os
 
 version = '2.9.1'
 
-setuptools.setup(
```

Comment:
This one can also be reverted if you do not pull setup into the namespace.

Reply:
yeah, as i said to @peterbarker i have no problem pulling this down if you didn't like it.

Reply:
Please do.